der/src/asn1/any.rs

```diff
@@ -294,3 +294,26 @@ where
         }
     }
 }
+
+#[cfg(feature = "alloc")]
+mod feature_alloc {
```

Comment:
recently discovered you could split implementation for trait in subpackages and decided it was the most appropriate time to make use of that ^^ (let me know if you just me to `#[cfg()]` everything instead)

Reply:
I like the idea, but maybe you could name it something like `allocating`

Reply:
<- bad at naming things